src/Momento.Sdk.Incubating/Requests/CollectionUpdateTtl.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+namespace Momento.Sdk.Incubating.Requests
+{
+    public record struct CollectionUpdateTtl(TimeSpan? ttl = null, bool refreshTtl = true)
+    {
+        public static CollectionUpdateTtl noRefreshOnUpdates()
+        {
```

Comment:
How about `noRefreshTtlOnUpdates`?

Reply:
Following this naming convention: https://github.com/momentohq/client-sdk-dotnet-incubating/pull/45/files#diff-d119a4cb0133e0bd2f2a05a6d109c9ed76443009f1634b5a72c61c4963c7d942R15